code/_onclick/hud/rendering/plane_master.dm

```diff
@@ -205,6 +205,11 @@
 	plane = RAD_TEXT_PLANE
 	render_relay_plane = RENDER_PLANE_NON_GAME
 
+/atom/movable/screen/plane_master/text_effect
+	name = "text effect plane"
+	plane = TEXT_EFFECT_PLANE
+	render_relay_plane = RENDER_PLANE_NON_GAME
```

Comment:
Would it be possible to combine this with the radtext plane to reduce the number of plane masters being drawn to before it is rendered to the screen